apps/outlook-rooms-addin/src/tests/test-mocks.ts

```diff
@@ -0,0 +1,340 @@
+import { FormBuilder, Validators } from '@angular/forms';
+import { of, Observable } from 'rxjs';
+import { MatBottomSheetRef } from '@angular/material/bottom-sheet';
+import { Space } from '@placeos/spaces';
+import { CalendarEvent } from '@placeos/events';
+
+export let mockForm = {
```

Comment:
I don't really think most of these mocks are necessary.
For the class instances you don't really gain anything from this over creating new instances of the live classes.
For the service mocks, it adds a layer of abstraction that I feel can make debugging and maintainability harder.